website/content/features.md

```diff
@@ -253,6 +249,10 @@ Always on the bleeding edge and built to last— Graphite is written on a robust
 			<img class="atlas" style="--atlas-index: 27" src="https://static.graphite.rs/icons/icon-atlas-roadmap__3.png" alt="" />
 			<span>Automation and batch processing</span>
 		</div>
+		<div class="feature-icon">
+			<img class="atlas" style="--atlas-index: 48" src="https://static.graphite.rs/icons/icon-atlas-roadmap__3.png" alt="" />
+			<span>Mesh vector format</span>
+		</div>
```

Comment:
Mesh vector data shouldn't be much work. We already have the vector data format. Tools don't need that much modification (they already work with the new format). The conversion to SVG is a bit harder but still not too complex if we accept improper stroke joins when >=3 lines converge.